example/ios/KlaviyoReactNativeSdkExample/Info.plist

```diff
@@ -20,8 +20,25 @@
 	<string>$(MARKETING_VERSION)</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
+	<key>CFBundleURLTypes</key>
```

Comment:
What is the change here?

Reply:
every time i run pod install it moves this, idk why